### PR TITLE
fix: change database database selector

### PR DESCRIPTION
**Pull request by pkdotson**
### SUMMARY
This pr fixes and issue where the database selector show an error toast when changing database. The issue come about when the params are undefined causing a method to error out.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
before

https://user-images.githubusercontent.com/17326228/152904976-bc74befd-0ff3-46ad-88a1-fa9e200bc371.mp4

after

https://user-images.githubusercontent.com/17326228/152906420-2c0ff7ab-f38c-4811-8a00-82a7d7ab9ea6.mov

### TESTING INSTRUCTIONS
Go to sqlab and click on database  selector. It should not give you an error toast when changing database.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by hughhhh — APPROVED** [review]

**Review by geido — APPROVED** [review]

**Comment by rosemarie-chiu** [comment]
🏷 preset:2022.5

File: superset-frontend/src/SqlLab/actions/sqlLab.js [hunk 1]
```diff
@@ -787,7 +787,11 @@ export function queryEditorSetSchema(queryEditor, schema) {
 
     return sync
       .then(() =>
-        dispatch({ type: QUERY_EDITOR_SET_SCHEMA, queryEditor, schema }),
+        dispatch({
+          type: QUERY_EDITOR_SET_SCHEMA,
+          queryEditor: queryEditor || {},
+          schema: schema || {},
```
Comment (john-bodley) [inline superset-frontend/src/SqlLab/actions/sqlLab.js:793]: @pkdotson or @hughhhh this seems wrong, i.e., schema is a nullable string and not an object. Should this be `schema: schema || null`?

Comment (villebro) [reply superset-frontend/src/SqlLab/actions/sqlLab.js:793]: @john-bodley you're right - here's the fix: #19664 (caught during testing of 1.5)

**Pull request merged**